src/benchmarking/tool_metrics/calculator.py

```diff
@@ -8,7 +8,16 @@
 
 
 class Calculator:
+    """
+    Class for run, evaluate (by compare with reference) and save results of llm's memory algorithms.
+    """
+
     def __init__(self, logger: BaseLogger, path_to_save: str | Path | None = None) -> None:
+        """
+        Initialization of Calculator class.
+        :param logger: the class for saving running and evaluating results.
+        :param path_to_save: path to save the result.
+        """
```

Comment:
1) please do not add comments that are evident from its arg names
2) why logger is "running and evaluating results"? I thought, it is a job for the `evaluator_function` and `logger` just logs?